package.json

```diff
@@ -56,7 +58,7 @@
     "@vue/cli-service": "4.2.3",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^4.0.0",
-    "@vue/test-utils": "1.0.0-beta.28",
+    "@vue/test-utils": "1.0.0-beta.33",
```

Comment:
Did you try the latest version?
https://www.npmjs.com/package/@vue/test-utils

It would be nice to graduate past the alpha/beta versions :)

(Yes, version 2 is in beta. I'm OK with staying on v1 for now.)

Reply:
I didn't try no, was content with a version i knew works well with the ported unit testing.
I will try 1.1.3 though, to see if we can officially graduate.

Reply:
Updated 👍